modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -66,7 +66,7 @@ if (!(process as any)?.browser) {
 
 const patchedRequestMethods = ['get', 'post', 'put', 'del', 'patch'] as const;
 
-export abstract class BitGoAPI implements BitGoBase {
+export abstract class AbstractBitGoAPI implements BitGoBase {
```

Comment:
can we just make this `export class BitGoAPI implements BitGoBase ` ? I don't think this class needs to be `abstract` after-all & would make things a bit simpler.

Reply:
Yeah, that works and compiles correctly.